src/FeeRouter/RewardReceiver.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.16;
+
+interface IArbSys {
+    function withdrawEth(
+        address destination
+    ) external payable returns (uint256);
+}
+
+contract RewardReceiver {
+    address immutable parentChainTarget;
+    uint256 immutable minDistributionIntervalSeconds;
+    uint256 public nextDistribution;
+
+    constructor(address _parentChainTarget,  uint256 _minDistributionIntervalSeconds) {
+        parentChainTarget = _parentChainTarget;
+        minDistributionIntervalSeconds = _minDistributionIntervalSeconds;
+    }
+
+    receive() external payable {
```

Comment:
need to add test to make sure the receive function is compatible with
https://github.com/OffchainLabs/fund-distribution-contracts/blob/80bea0f028be08d8f37d961569c7c2d8f4cca96c/src/RewardDistributor.sol#L22
i.e. use less than 100k gas

Reply:
And if we decided to skip the RewardDistributor instead, since l2 fees appear in the contract like CL reward without evm execution, someone will need to trigger this receive function manually.

Reply:
> And if we decided to skip the RewardDistributor instead, since l2 fees appear in the contract like CL reward without evm execution, someone will need to trigger this receive function manually.
could just have receive call a public function